src/libUtils/SysCommand.h

```diff
@@ -17,17 +17,47 @@
 #ifndef __SYSCOMMAND_H__
 #define __SYSCOMMAND_H__
 
+#include <array>
 #include <cstdlib>
 #include <fstream>
 #include <iostream>
+#include <memory>
 #include <string>
 
-void ExecuteCmd(const std::string cmd)
+const std::string EXEC_CMD_LOG = "ExecuteCmd.txt";
+
+class SysCommand
 {
-    int ret = std::system((cmd + " > ExecuteCmd.txt").c_str());
-    (void)ret;
-    LOG_GENERAL(INFO,
-                "ExecuteCmd: " << std::ifstream("ExecuteCmd.txt").rdbuf());
-}
+public:
+    static void ExecuteCmd(const std::string cmd)
```

Comment:
If this function `ExecuteCmd` is desired to run without output (as opposed to `ExecuteCmdWithOutput`), why don't we just drop stdout and stderr totally?